packages/mobile/src/screens/chat-screen/ChatListScreen.tsx

```diff
@@ -1,42 +1,94 @@
 import { useEffect } from 'react'
 
 import { chatActions, chatSelectors, Status } from '@audius/common'
-import { View } from 'react-native'
+import { View, Text } from 'react-native'
 import { useDispatch, useSelector } from 'react-redux'
 
 import IconCompose from 'app/assets/images/iconCompose.svg'
 import IconMessage from 'app/assets/images/iconMessage.svg'
+import Button, { ButtonType } from 'app/components/button'
 import { Screen, FlatList, ScreenContent } from 'app/components/core'
 import LoadingSpinner from 'app/components/loading-spinner'
 import { useNavigation } from 'app/hooks/useNavigation'
 import type { AppTabScreenParamList } from 'app/screens/app-screen'
 import { makeStyles } from 'app/styles'
-import { useThemePalette } from 'app/utils/theme'
+import { useThemePalette, useColor } from 'app/utils/theme'
 
 import { ChatListItem } from './ChatListItem'
 
 const { getChats, getChatsStatus } = chatSelectors
 
 const messages = {
-  title: 'Messages'
+  title: 'Messages',
+  startConversation: 'Start a Conversation!',
+  connect:
+    'Connect with other Audius users by\nstarting a private direct message!',
+  writeMessage: 'Write a Message'
 }
 
 const useStyles = makeStyles(({ spacing, palette, typography }) => ({
   rootContainer: {
     display: 'flex',
-    justifyContent: 'center',
     flexGrow: 1
   },
   spinnerContainer: {
-    height: spacing(28)
+    height: spacing(28),
+    padding: spacing(10)
   },
   loadingSpinner: {
     height: spacing(20),
     width: spacing(20),
     alignSelf: 'center'
+  },
+  startConversationContainer: {
+    marginVertical: spacing(8),
+    marginHorizontal: spacing(4),
+    backgroundColor: palette.white,
+    borderWidth: 1,
+    borderColor: palette.neutralLight7,
+    borderRadius: spacing(2),
+    padding: spacing(6)
+  },
+  startConversationTitle: {
+    fontSize: typography.fontSize.xxl,
+    fontFamily: typography.fontByWeight.bold,
+    textAlign: 'center',
+    lineHeight: 31,
+    color: palette.neutral
+  },
+  connect: {
+    fontSize: typography.fontSize.medium,
+    textAlign: 'center',
+    lineHeight: 21,
```

Comment:
For these it might make sense to make them multiples of the font size, eg `typography.fontSize.medium * 1.3` or whatever it is. What do you think?